Subject: Fix parallel build issue
Origin: upstream, commit 1.10.0-2-g5269b57
Author: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Fri May 17 22:25:52 2019 +0200

    The config.h header file must be created before being used to build
    source files that include it, especially version.c. In order for this
    to happen even in highly-parallel builds, we add a dependency of all
    object files on config.h, in order to ensure it gets generated before
    make attempts to build the object files.

    Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
    [Retrieved from:
    https://git.buildroot.net/buildroot/tree/package/pptp-linux/0002-fix-parallel-build.patch]
    Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>

--- a/Makefile
+++ b/Makefile
@@ -42,6 +42,8 @@
 
 all: config.h $(PPTP_BIN) pptpsetup.8
 
+$(PPTP_OBJS): config.h
+
 $(PPTP_BIN): $(PPTP_OBJS) $(PPTP_DEPS)
 	$(CC) -o $(PPTP_BIN) $(PPTP_OBJS) $(LDFLAGS) $(LIBS)
 
